modules/KalturaSupport/components/live/liveCore.js

```diff
@@ -95,6 +95,11 @@
 					vid.removeEventListener('progress', _this.onProgress);
 					_this.onProgress = null;
 				}
+                if (_this.offAirTimeout){
```

Comment:
add a new api to reset `this.offAirTimeout` and use it here and in the rest places we reset it.